cmake/AddResources.cmake

```diff
@@ -61,6 +64,14 @@ function(add_resources target)
 				"${android_dir}/assets/${file}"
 				SYMBOLIC
 			)
+		# Assets will already be installed by MACOSX_PACKAGE_LOCATION on macOS
+		elseif(NOT APPLE)
+			# No trailing slash is required or else it will strip the last path
+			# of the directory
```

Comment:
This comment doesn’t actually apply to `install(FILES)`, since the problem of trailing slashes has to do with the source of an `install(DIRECTORY)`.

Reply:
Purged the comment.